src/api/VoteMonitor.Api.Observer/Controllers/ObserverController.cs

```diff
@@ -42,17 +42,59 @@ public async Task<List<ObserverModel>> GetObservers(ObserverListQuery query)
         public void Import(IFormFile file, [FromForm] object a)
         { }
 
+        /// <summary>
+        ///  Adds an observer.
+        /// </summary>
+        /// <param name="model"></param>
+        /// <returns>Boolean indicating whether or not the observer was added successfully.</returns>
         [Authorize]
```

Comment:
```suggestion
```

the `Authorize` attribute should be implicit on all actions. Was this misbehaving? Did we break anything when refactoring auth? (we specified `AllowAnonymous` for the necessary routes until now)

Reply:
Nope, don't know why that was there :)